<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PiCam</class>
 <widget class="QWidget" name="PiCam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="12,2,1">
   <item>
    <widget class="QGraphicsView" name="pic"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="int_LB">
       <property name="text">
        <string>interval/s</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="int_LE"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="int_BTN">
       <property name="text">
        <string>update</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ang_LB">
       <property name="text">
        <string>angle/deg</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ang_LE"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="ang_BTN">
       <property name="text">
        <string>update</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="fetch">
       <property name="text">
        <string>fetch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="warning">
       <property name="text">
        <string>warning</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
